site/webpack-client.config.js

```diff
@@ -14,7 +15,7 @@ module.exports = {
 
   output: {
     path: path.resolve(__dirname, '../__site__/'),
-    filename: isDev ? '[name].js' : '[name]-[hash].js',
+    filename: isDev ? '[name].js' : '[name]-[contentHash].js',
```

Comment:
Webpack recommends using `contentHash` instead of `hash`.
> When using [hash] placeholder in webpack configuration, consider changing it to [contenthash]. It is not the same, but proven to be more effective.